docs/api-spec.md

```diff
@@ -351,6 +327,72 @@ token needs to be presented when creating a replica of this primary account.
 Sublease tokens can only be issued for primary accounts. If the account in question is a replica account, 400 (Bad
 Request) is returned.
 
+## GET /keppel/v1/accounts/:name/trivy\_policies
```

Comment:
Why did we name this trivy_polices and not security_policies or similar? It is not really deeply tied into trivy.

Reply:
I agree on concern of tying the name too closely to the backing service. However, "security policy" is too vague in my opinion. I'll change it to "security scan policy".